docs/devel/development.md

```diff
@@ -339,6 +354,8 @@ hack/update-generated-docs.sh
 ```
 
 
+
```

Comment:
@bgrant0607 That's a pretty bad sign. The process shouldn't be so complex that it prevents us from doing changes ..

I'll add mentions in the LGTM message right now.

The do-no-merge is a bug that appeared earlier this week: https://github.com/kubernetes/contrib/issues/1445